### PR TITLE
ID-294 Update scala-library to 2.13.10

### PR DESCRIPTION
**Pull request by broadbot**
Updates [org.scala-lang:scala-library](https://github.com/scala/scala) from 2.13.5 to 2.13.10.
[GitHub Release Notes](https://github.com/scala/scala/releases/tag/v2.13.10) - [Version Diff](https://github.com/scala/scala/compare/v2.13.5...v2.13.10)

I'll automatically update this PR to resolve conflicts as long as you don't change it yourself.

If you'd like to skip this version, you can just close this PR. If you have any feedback, just mention me in the comments below.

Configure Scala Steward for your repository with a [`.scala-steward.conf`](https://github.com/scala-steward-org/scala-steward/blob/e93966d6d80e4ff1b31567984aa7e70e11cb2070/docs/repo-specific-configuration.md) file.

Have a fantastic day writing Scala!

<details>
<summary>Files still referring to the old version number</summary>

The following files still refer to the old version number (2.13.5).
You might want to review and update them manually.
```
codegen_java/templates/libraries/okhttp-gson/build.sbt.mustache
```
</details>
<details>
<summary>Adjust future updates</summary>

Add this to your `.scala-steward.conf` file to ignore future updates of this dependency:
```
updates.ignore = [ { groupId = "org.scala-lang", artifactId = "scala-library" } ]
```
Or, add this to slow down future updates of this dependency:
```
dependencyOverrides = [{
  pullRequests = { frequency = "@monthly" },
  dependency = { groupId = "org.scala-lang", artifactId = "scala-library" }
}]
```
</details>
<details>
<summary>Note that the Scala Steward config file `.scala-steward.conf` wasn't parsed correctly</summary>

```
cron4s.InvalidCron: Either DayOfMonth and DayOfWeek must have a ? expression: DownField(frequency),DownField(pullRequests)
```
</details>

labels: library-update, early-semver-patch, semver-spec-patch, old-version-remains, commit-count:1

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/workbench/sam/db/DbReference.scala [hunk 1]
```diff
@@ -14,8 +14,8 @@ import org.broadinstitute.dsde.workbench.sam.util.SamRequestContext
 import org.broadinstitute.dsde.workbench.util2.ExecutionContexts
 import scalikejdbc.config.DBs
 import scalikejdbc.{ConnectionPool, DBSession, IsolationLevel, NamedDB}
-import sun.security.provider.certpath.SunCertPathBuilderException
 
+import java.security.cert.CertPathBuilderException
```
Comment (tlangs) [inline src/main/scala/org/broadinstitute/dsde/workbench/sam/db/DbReference.scala:18]: https://www.oracle.com/java/technologies/faq-sun-packages.html

**Comment by tlangs** [comment]
Addresses https://github.com/broadinstitute/sam/pull/844

**Comment by tlangs** [comment]
jenkins retest

**Comment by tlangs** [comment]
jenkins retest

**Comment by tlangs** [comment]
jenkins retest

**Comment by tlangs** [comment]
jenkins retest

**Comment by tlangs** [comment]
jenkins retest

**Review by Shakespeared — APPROVED** [review]

**Review by gpolumbo-broad — APPROVED** [review]

**Pull request merged**